app/src/main/java/com/mychefassistant/presentation/kitchen/insert/KitchenInsertFragment.kt

```diff
@@ -73,6 +101,16 @@ class KitchenInsertFragment : Fragment() {
             .show()
 
     private fun routeToGrocery(id: Int) = findNavController().navigate(
-        R.id.action_kitchen_insert_to_grocery_manage2, bundleOf("id" to id)
+        R.id.action_fragment_kitchen_insert_to_fragment_grocery_manage, bundleOf("id" to id)
     )
+
+    companion object {
+        val icons = arrayOf(

```

Comment:
Are you using these everywhere? Why are they public?
Why are they static? Are you using them in any static context?